pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -35,6 +35,21 @@ let
   audio = lib.optionalString alsaSupport "alsa,"
     + lib.optionalString pulseSupport "pa,"
     + lib.optionalString sdlSupport "sdl,";
+  emit-kvm-warnings = writeShellScriptBin "emit-kvm-warnings" ''
+    WARNCOL='\033[0m'
+    NEUTRALCOL='\033[0m'
```

Comment:
I don't think these two definitions are supposed to be the same.

Reply:
Thanks, I don't know how that had happened. d033935ab6fd850160e31a1db822486f39650210